src/get-started/create-service.md

```diff
@@ -492,13 +517,11 @@ With this aim we declare a blank struct that includes a set of methods with the
 following signature:
 
 ```rust
-fn my_method(state: &ServiceApiState, query: MyQuery) -> api::Result<MyResponse>
+fn my_method(state: &ServiceApiState<'_>, query: MyQuery) -> api::Result<MyResponse>
```

Comment:
Neither of two methods described below satisfies this signature 🙃. I'd propose to remove this sentence and explain what `state` is after the corresponding code sample.